test/TestLinearPremiumPriceOracle.js

```diff
@@ -0,0 +1,63 @@
+const ENS = artifacts.require('@ensdomains/ens/ENSRegistry');
+const BaseRegistrar = artifacts.require('./BaseRegistrarImplementation');
+const DummyOracle = artifacts.require('./DummyOracle');
+const LinearPremiumPriceOracle = artifacts.require('./LinearPremiumPriceOracle');
+
+const namehash = require('eth-ens-namehash');
+const sha3 = require('web3-utils').sha3;
+const toBN = require('web3-utils').toBN;
+
+contract('LinearPremiumPriceOracle', function (accounts) {
+    let priceOracle;
+
+    before(async () => {
+		ens = await ENS.new();
+		registrar = await BaseRegistrar.new(ens.address, namehash.hash('eth'));
+		await ens.setSubnodeOwner('0x0', sha3('eth'), registrar.address);
+		await registrar.addController(accounts[0]);
```

Comment:
Please fix indentation

Reply:
I think you are not seeing these indentations in your editor as you set indentation space to 4. Change it to 2 and you will see it as we see on github